main.go

```diff
@@ -72,6 +73,26 @@ func runCommand() error {
 		return nil
 	}
 
+	// read tsa and include-certs from gitconfig
+	path, err := os.Getwd()
+	if err == nil {
+		repo, err := git.OpenRepository(path)
+		if err == nil {
+			config, err := repo.Config()
+
+			tsa, err := config.LookupString("gpg.x509.smimesign.timestamp-authority")
```

Comment:
What do we get from `LookupString` if the config value isn't set? Do we get an empty string (i.e. we match the default value)?